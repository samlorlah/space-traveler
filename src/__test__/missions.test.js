import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import Mission from '../components/mission';

it('mission component test', () => {
  const component = renderer.create(
    <Provider store={store}>
      <Mission />
    </Provider>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
