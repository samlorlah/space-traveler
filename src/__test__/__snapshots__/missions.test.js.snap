// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mission component test 1`] = `
<div
  className="tableContainer"
>
  <table
    className="table"
  >
    <thead
      className="head"
    >
      <tr>
        <th>
          Name
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
        <th>
           
        </th>
      </tr>
    </thead>
    <tbody
      className="body"
    />
  </table>
</div>
`;
